bundles/org.eclipse.swt/Eclipse SWT/common/org/eclipse/swt/internal/SVGRasterizerRegistry.java

```diff
@@ -0,0 +1,39 @@
+/*******************************************************************************
+ * Copyright (c) 2025 Vector Informatik GmbH and others.
+ *
+ * This program and the accompanying materials are made available under the terms of the Eclipse
+ * Public License 2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors: Michael Bangas (Vector Informatik GmbH) - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.swt.internal;
+
+import java.util.*;
+
+/**
+ * A registry for managing the instance of an {@link SVGRasterizer}
+ * implementation. This allows for the registration and retrieval of a single
+ * rasterizer instance.
+ *
+ * @since 3.129
+ */
+public class SVGRasterizerRegistry {
+
+	/**
+	 * The instance of the registered {@link SVGRasterizer}.
+	 */
+	private static final SVGRasterizer RASTERIZER = ServiceLoader.load(SVGRasterizer.class).findFirst().orElse(null);
```

Comment:
Since this class now basically just holds the reference to the `SVGRasterizer` and it's getter is only called once, I suggest to move this into the `FileFormat` class and delete this class.